bin/improver

```diff
@@ -58,6 +58,6 @@ fi
 
 # Put our library and scripts in the paths.
 export PATH="$IMPROVER_DIR/bin/:$PATH"
-export PYTHONPATH="$IMPROVER_DIR/lib/:${PYTHONPATH:-}"
+export PYTHONPATH="$IMPROVER_DIR/:${PYTHONPATH:-}"
```

Comment:
> Test imports have been changed to relative imports in order to avoid "import tests" which is a very generic name and at risk of namespace conflicts.

I still wonder if leaving this directory named as `tests` is a good idea. It's all too easy for it to end up on `PYTHONPATH` and my quick google turned up some reports about this causing breakage. I think something like `improver_tests` would be much safer (and with relative imports we won't even see that name much around anyway).

Reply:
> > Test imports have been changed to relative imports in order to avoid "import tests" which is a very generic name and at risk of namespace conflicts.
> 
> I still wonder if leaving this directory named as `tests` is a good idea. It's all too easy for it to end up on `PYTHONPATH`

I don't know how I feel about this. I think having the directory named `tests` is cleaner, and I kind of feel like if people are putting things in their PYTHONPATH that shouldn't be there, then that's their own fault... But maybe I'm just less forgiving than other people. 😅

Reply:
I think it's not unreasonable to want `improver` straight from git checkout on `PYTHONPATH`, there are valid use cases for this. But that unfortunately will put `tests` there too. Why having `tests` is cleaner? Is it worth the risk of name conflicts?

Reply:
> I think it's not unreasonable to want `improver` straight from git checkout on `PYTHONPATH`, there are valid use cases for this.

Hmm... I guess that's fair, but then you'd also be including `bin` and the README etc in the PYTHONPATH as well - which might be harmless (I haven't checked), but also seems a bit messy.

> Why having `tests` is cleaner?

🤷‍♂  It just feels cleaner, but I guess that's just because I consider the repo itself as a sort of namespacing. Maybe this would go away if we had `tests` as a subdirectory inside `improver`.....

> Is it worth the risk of name conflicts?

Probably not, no. I'm not that strongly against `improver_tests` if we think a bare `tests` directory is asking for trouble.

Reply:
Another possible name: `test_impover`, sort of like the individual test name, but for the whole package.

Reply:
> Another possible name: `test_improver`, sort of like the individual test name, but for the whole package.

That's actually quite nice. I could get behind that. @tjtg, what are your thoughts?

Reply:
(Discussed with @tjtg. We're going with `improver_tests`)

Reply:
I've added commits to rename from `tests` to `improver_tests`.